src/app/server/Server.h

```diff
@@ -38,7 +38,11 @@
 #include <credentials/OperationalCertificateStore.h>
 #include <credentials/PersistentStorageOpCertStore.h>
 #include <crypto/OperationalKeystore.h>
+#ifdef ENABLE_HSM_EC_KEY
+#include <crypto/hsm/nxp/PersistentStorageOperationalKeystoreHSM.h>
```

Comment:
Shouldn't this get handled by applications instead of modifying the deprecated InitializeStaticResourcesBeforeServerInit?

Reply:
Removed from InitializeStaticResourcesBeforeServerInit. Will be moved to app specific file.